### PR TITLE
done including test

### PR DESCRIPTION
**Pull request by silasoyewale10**

### CODE REVIEW

File: Data-Structures/src/test/java/datastructures/tree/FizzBuzzTreeTest.java [hunk 1]
```diff
@@ -36,4 +36,34 @@
...
     }
 
+    @Test public void testTraversal() {
+        //       15
+        //    15     7
+        //          9  10
+//        NodeString root =
+        NodeString root = new NodeString("15", new NodeString("15"), new NodeString("7", new NodeString("9"), new NodeString("10")));
+        FizzBuzzTree tree = new FizzBuzzTree(root);
+        assertEquals("[15, 15, 7, 9, 10]",tree.breadthPrint().toString() );
```
Comment (jeffborda) [inline Data-Structures/src/test/java/datastructures/tree/FizzBuzzTreeTest.java:46]: Remember, try to avoid testing with `toString()` unless you are testing `toString()`. This test could be refactored like this:  `assertArrayEquals(new Integer[]{15, 15, 7, 9, 10}, tree.breadthPrint().toArray());`

File: Data-Structures/src/test/java/datastructures/tree/FizzBuzzTreeTest.java [hunk 2]
```diff
@@ -36,4 +36,34 @@
...
+
+    }
+    @Test public void testTraversalThird() {
+
+        NodeString root3 = new NodeString("3");
+        FizzBuzzTree tree = new FizzBuzzTree(root3);
+
+        System.out.println(tree.breadthPrint());
+        assertEquals("[3]", tree.breadthPrint().toString());
+    }
```
Comment (jeffborda) [inline Data-Structures/src/test/java/datastructures/tree/FizzBuzzTreeTest.java:67]: I like that you are thinking about edge cases, like a tree with only one node 👍

File: Data-Structures/src/main/java/datastructures/tree/FizzBuzzTree.java [hunk 3]
```diff
@@ -25,5 +30,31 @@ else if(runnerValue % 3 == 0) {       //check if input is greater than
             replace(runner.right);
         }
     }
+    public ArrayList breadthPrint() {
```
Comment (jeffborda) [inline Data-Structures/src/main/java/datastructures/tree/FizzBuzzTree.java:33]: Your algorithm looks great, but I don't like the name "breadthPrint" because you are not actually printing anything, you're returning an array list in breadth first order.
